website/docs/r/sagemaker_model.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "aws"
+page_title: "AWS: sagemaker_model"
+sidebar_current: "docs-aws-resource-sagemaker-model"
+description: |-
+  Provides a SageMaker model resource.
+---
+
+# aws\_sagemaker\_model
```

Comment:
Backslashes in resource documentation titles were removed from the codebase awhile ago. Presumably this PR predates that cleanup.